styles.css

```diff
@@ -152,11 +147,18 @@
     margin-right: auto;
     margin-left: auto;
   }
-  .filter{
+  .filters {
+    margin-bottom: 10px;
+    display: none;
+  }
+  
+  .filters-open {
     display: flex;
     justify-content: center;
   }
-  .body{
+  
+  }
```

Comment:
This needs to be removed then its all working

Reply:
done and pushed